daprdocs/content/en/reference/components-reference/supported-pubsub/_index.md

```diff
@@ -32,6 +32,11 @@ Table captions:
 | [RabbitMQ]({{< ref setup-rabbitmq.md >}})             | Beta  | v1 | 1.6 |
 | [Redis Streams]({{< ref setup-redis-pubsub.md >}})    | Stable | v1 | 1.0 |
 
+### Alibaba Cloud
+| Name                                              | Status | Component version | Since |
+|---------------------------------------------------|--------| ---- |---------------|
+| [Alibaba Cloud RocketMQ]({{< ref setup-alicloud-rocketmq.md >}}) | Alpha | v1 | 1.8 |
```

Comment:
This should not be under Alibaba. This should be under generic. Even the type says type: pubsub.rocketmq.

Reply:
Got it. This PR was approved earlier as an Alibaba specific component https://github.com/dapr/docs/pull/1405. Moving this to a generic one.